src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/NewEditCyclePage/PayoutsSection/PayoutsTable/modals/SwitchToUnlimitedModal.tsx

```diff
@@ -12,10 +14,12 @@ export function SwitchToUnlimitedModal({
   onClose: VoidFunction
 }) {
   const { editCycleForm } = useEditCycleFormContext()
+  const { payoutSplits } = usePayoutsTable()
 
   const onOk = () => {
     editCycleForm?.setFieldsValue({
       distributionLimit: undefined,
+      payoutSplits: ensureSplitsSumTo100Percent({ splits: payoutSplits }),
```

Comment:
Also noticed a weird bug when switching to unlimited sometimes. Should always be exactly 100% in that case

Reply:
nice, fair lol